mmdet/models/dense_heads/fcos_head.py

```diff
@@ -109,31 +133,50 @@ def _init_layers(self):
 
     def init_weights(self):
         for m in self.cls_convs:
-            normal_init(m.conv, std=0.01)
+            if isinstance(m.conv, nn.Conv2d):
+                normal_init(m.conv, std=0.01)
         for m in self.reg_convs:
-            normal_init(m.conv, std=0.01)
+            if isinstance(m.conv, nn.Conv2d):
+                normal_init(m.conv, std=0.01)
         bias_cls = bias_init_with_prob(0.01)
         normal_init(self.fcos_cls, std=0.01, bias=bias_cls)
         normal_init(self.fcos_reg, std=0.01)
         normal_init(self.fcos_centerness, std=0.01)
 
     def forward(self, feats):
-        return multi_apply(self.forward_single, feats, self.scales)
+        return multi_apply(self.forward_single, feats, self.scales,
+                           self.strides)
+    """
+    Args:
+        x (torch.Tensor): FPN feature maps of the specified stride
+        scale (mmcv.cnn.Scale): Factor to resize the bbox prediction.
```

Comment:
scale (:obj: \`mmcv.cnn.Scale\`): Learnable scale module to resize the bbox prediction.